### ISSUE TITLE
Component Accessibility Status: EAC Default state baseline audit/coverage

### ISSUE THREAD

**Issue by laurenmrice** [carbon-design-system/carbon#13746]
### Acceptance criteria

- [x] There are no violations when running the automated [Equal Access Checker](https://www.ibm.com/able/toolkit/tools) on default component state.
- [x] Make sure we have the default state baselines for all the components, especially any new components or enhancement to existing components since Josh [first implemented the EAC](https://github.com/carbon-design-system/carbon/issues/9475).

```[tasklist]
### Components missing avt in e2e files
- [x] ClassPrefix
- [x] ErrorBoundary
- [x] FluidForm
- [x] Grid
- [x] Heading
- [x] IconButton
- [x] IdPrefix
- [x] OverFlowMenuV2
```

These two components are waiting changes on this [issue](https://github.com/IBMa/equal-access/issues/761)
- [ ] Slider
- [ ] Toggle
